technology/design-considerations.md

```diff
@@ -78,12 +78,10 @@ Because sequencing maintains no application state, it can be scaled out by addin
 
 Security of critical protocols like Distributed Key Generation (DKG) and Threshold Signature Schemes (TSS) requires a large, diverse validator set:
 
-- Pelagos targets ~100+ restaking operators initially, scaling toward 500+ for long‑term resilience.
+- Pelagos targets ~100+ POS operators initially, scaling toward 500+ for long‑term resilience.
```

Comment:
@JekaMas I am assuming more emphasis needed on the PoS given the removal of restaking --> Pls can you bring the actual numbers for you minimum validator set